config-connector/solutions/iam/helm/service-account/templates/iampolicymember.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: iampolicymember-service-account
+spec:
+  member: {{ required "IAM member is required!" .Values.iamPolicyMember.iamMember }}
```

Comment:
I'm hesitating whether we want to strictly follow how kpt solutions implement required fields. Let's talk about this.

Reply:
I don't think we need to follow it very strictly as long as we follow the structure and extent of customization: same configuration is produced, only setters and same setters.

Reply:
But looks like the iam member is always required. What default value could we set for it?

Reply:
Yes, iam member is always required. So feel free to use a placeholder or leave it empty like what you've already done.
Agreed with @AlexBulankou that following the kpt solutions strictly is not necessary so feel free to use Helm's best practice (if there is any) for required fields.

Reply:
Make the IAM member to be required.